src/js/components/PackageDetailVersionsDropdown.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+
+import { FormGroup, FieldSelect } from "#SRC/js/components/form";
+
+const METHODS_TO_BIND = ["handleVersionChange"];
+
+class PackageDetailVersionsDropdown extends Component {
+  constructor() {
+    super();
+
+    METHODS_TO_BIND.forEach(method => {
+      this[method] = this[method].bind(this);
+    });
+  }
+
+  handleVersionChange(event) {
```

Comment:
Here I use the mechanism already in place which is based on the query `version` so I only change the location that way triggering the rest of the actions.